assets/app/styles/_core.less

```diff
@@ -4,6 +4,7 @@
 
 html, body {
   height: 100%;
+  word-wrap: break-word;
```

Comment:
@jcantrill 
This was included in my earlier pull request as a legacy carry over from V2, but let's remove it and apply that on a per case basis.